ui/public/checkBrowser.js

```diff
@@ -0,0 +1,27 @@
+function checkBrowser() {
+  if (!window.__supported_browsers__.test(navigator.userAgent)) {
+    var text
+    if (navigator.language.indexOf('zh') === 0) {
+      text =
+        '您的浏览器版本已过期，推荐使用最新版本的 Chrome 或 Edge 浏览器以便获得更好的体验。'
+    } else {
+      text =
+        'Your browser version is outdated, recommend to use the latest Chrome or Edge to get better experience.'
+    }
+
+    const content =
+      '<div style="background: yellow; width: 100%; position: absolute; top: 0; z-index: 4; padding: 8px; text-align: center; transition: top 2s;">' +
```

Comment:
How about using red as the background to bring more attractions

Reply:
I am ok with it, just found some examples which use the yellow background, so do I.

![image](https://user-images.githubusercontent.com/1284531/84724852-be9e1e00-afbb-11ea-8fe9-37f2db460fe1.png)

![image](https://user-images.githubusercontent.com/1284531/84724895-df667380-afbb-11ea-9ffa-62461abb2bc0.png)

Reply:
How about two policies:

- For browsers that functionality will totally break, must not continue, asking user to change browser, like:
  ![image](https://user-images.githubusercontent.com/1916485/84729596-b4355180-afc6-11ea-930d-e868c2856a02.png)
- For browsers that functionality works (but maybe not perfect), display a soft recommendation.

Reply:
Compared following colors (choose from ant design colors):

red:
![red](https://user-images.githubusercontent.com/1284531/84733948-3ecf7e00-afd2-11ea-88b9-f1283ec49973.png)

ff4d4f:
![#ff4d4f](https://user-images.githubusercontent.com/1284531/84733965-4858e600-afd2-11ea-8f2e-c59e1b721a23.png)

yellow:
![yellow](https://user-images.githubusercontent.com/1284531/84733983-573f9880-afd2-11ea-9da1-144da14cc317.png)

ffc53d:
![#ffc53d](https://user-images.githubusercontent.com/1284531/84734002-60c90080-afd2-11ea-86bf-10c23a3fff6b.png)

fadb14:
![#fadb14](https://user-images.githubusercontent.com/1284531/84734015-6cb4c280-afd2-11ea-8834-9723d7c1b86e.png)

Feel the last one makes me most comfortable.

Reply:
> How about two policies:
> 
> * For browsers that functionality will totally break, must not continue, asking user to change browser, like:
>   ![image](https://user-images.githubusercontent.com/1916485/84729596-b4355180-afc6-11ea-930d-e868c2856a02.png)
> * For browsers that functionality works (but maybe not perfect), display a soft recommendation.

This would be better.